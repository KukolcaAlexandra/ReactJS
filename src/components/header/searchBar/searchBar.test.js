import React from 'react';
import SearchBar from './searchBar.component';
import {shallow} from 'enzyme';

describe('SearchBar component', () => {
    
  it('should be render correctly with Title', () => {
    const onClickSearchButton = jest.fn();
    const onClickFilterButton = jest.fn();
    const searchBy = 'TITLE';

    const component = shallow(
      <SearchBar
        onClickSearchButton={onClickSearchButton}
        onClickFilterButton={onClickFilterButton}
        searchBy={searchBy}
      />
    );
    expect(component).toMatchSnapshot();
  });  
  
  it('should be render correctly with Genre', () => {
    const onClickSearchButton = jest.fn();
    const onClickFilterButton = jest.fn();
    const searchBy = 'GENRE';

    const component = shallow(
      <SearchBar
        onClickSearchButton={onClickSearchButton}
        onClickFilterButton={onClickFilterButton}
        searchBy={searchBy}
      />
    );
    expect(component).toMatchSnapshot();
  });  
  
})