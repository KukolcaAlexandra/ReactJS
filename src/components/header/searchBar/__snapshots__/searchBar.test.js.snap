// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar component should be render correctly with Genre 1`] = `ShallowWrapper {}`;

exports[`SearchBar component should be render correctly with Title 1`] = `ShallowWrapper {}`;
